syncer/ddl.go

```diff
@@ -239,6 +239,11 @@ func (s *Syncer) dropSchemaInSharding(sourceSchema string) error {
 	sources := make(map[string][][]string)
 	sgs := s.sgk.Groups()
 	for name, sg := range sgs {
+		if sg.IsSchemaOnly {
+			// in sharding group leave handling, we always process schema group,
```

Comment:
```suggestion
			// in sharding group leave handling, we always don't process schema group,
```
?

Reply:
https://github.com/pingcap/dm/blob/eabc76e625b9a67e37dc3c1493824bf2d690435a/syncer/sharding_group.go#L419-L437
`process schema group` means we remove sources pair for schema group here

Reply:
lol, so long a way